code/modules/admin/secrets.dm

```diff
@@ -546,6 +547,23 @@
 				B.update_hair()
 			message_admins("[key_name_admin(usr)] activated dorf mode")
 
+		if("schoolgirl")
+			if(!check_rights(R_FUN))
+				return
+			feedback_inc("admin_secrets_fun_used",1)
+			feedback_add_details("admin_secrets_fun_used","SG")
+			for(var/obj/item/clothing/under/W in world)
```

Comment:
correct me if I'm wrong but I believe that in world actually has special handling depending on what you're looking for. I mean it is a bit laggy all the same, but for example I don't think it'll compare turfs against undershirts. Seems a bit of a waste to make a global list to be maintained every single round for a badmin only button that might get pressed once every five hundred rounds.